.github/workflows/pr-check-e2e.yml

```diff
@@ -47,3 +47,4 @@ jobs:
           NEXT_PUBLIC_AUTH_REALM: 'ien'
           NEXT_PUBLIC_AUTH_CLIENTID: 'IEN'
           NEXT_PUBLIC_AUTH_URL: 'https://keycloak.freshworks.club/auth'
+          NEXT_PUBLIC_DISABLE_USER_GUIDE: 'true'
```

Comment:
When running this locally with the user guide disabled, the admin page will get in a loop of requesting the userguide, then displaying a 500 error stating the user guide is disabled, then re-querying for it. It's a bit jarring. If the user guide is disabled, when fetching it, can we instead return an empty array.

Reply:
If NEXT_PUBLIC_DISABLE_USER_GUIDE is set, the admin page should not load the user-guide component and there should be no call to fetch user guides. Have you fully refreshed the page after disabling it?